java/core/src/test/org/apache/orc/TestOrcDSTNoTimezone.java

```diff
@@ -34,8 +34,8 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 
-import static junit.framework.Assert.assertEquals;
-import static junit.framework.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.assertFalse;
```

Comment:
Yes, this looks consistent and better than before because we mixed `org.junit` and `junit.framework` previously.